### ISSUE TITLE
Doesn't seem to work for me (Win10 1803)

### ISSUE THREAD

**Issue by Derkades** [KazeJiyu/eclipse-discord-integration#56]
Added game manually in game activity menu, does show up but no RP
![image](https://user-images.githubusercontent.com/15892014/48728462-4a5f9500-ec35-11e8-8d32-9a493c9256b3.png)

Using the default settings
![image](https://user-images.githubusercontent.com/15892014/48728485-5cd9ce80-ec35-11e8-8308-87ea4d439c81.png)

Enabled in startup
![image](https://user-images.githubusercontent.com/15892014/48728499-66fbcd00-ec35-11e8-962e-bcb226f0400b.png)

Using eclipse Oxygen
![image](https://user-images.githubusercontent.com/15892014/48728518-72e78f00-ec35-11e8-8264-415dc6974689.png)

Windows 10 1803
![image](https://user-images.githubusercontent.com/15892014/48728565-90b4f400-ec35-11e8-9498-1dd0295dfaac.png)

**Comment by Clem-Fern** [comment]
Same for me with Eclipse 2018-09 (4.9.0) Build id: 20180917-1800 and Windows 10 (1803)

**Comment by KazeJiyu** [comment]
I cannot reproduce this issue: on my Windows 10 the plug-in works with both Eclipse IDE 4.7 and 4.9. Could you open the _Error Log_ view (`Window > Show View > Other... > Error Log`) and check whether any log seems related to the plug-in ? The name in the `Plug-in` column should start with `fr.kazejiyu.discord`.

**Comment by Clem-Fern** [comment]
Nothing related with your plugin in my error log.

**Comment by Derkades** [comment]
I see the following error in my error log from `fr.kazejiyu.discord.rpc.integration`:
```
org.eclipse.core.internal.resources.ResourceException: Project 'Discord Bot' is not open.
	at org.eclipse.core.internal.resources.Project.checkAccessible(Project.java:144)
	at org.eclipse.core.internal.resources.Resource.checkAccessibleAndLocal(Resource.java:215)
	at org.eclipse.core.internal.resources.Resource.getPersistentProperty(Resource.java:1125)
	at fr.kazejiyu.discord.rpc.integration.ui.preferences.properties.ProjectPropertiesPage.setMissingPropertiesToDefault(ProjectPropertiesPage.java:131)
	at fr.kazejiyu.discord.rpc.integration.ui.preferences.properties.ProjectPropertiesPage.createContents(ProjectPropertiesPage.java:84)
	at org.eclipse.jface.preference.PreferencePage.createControl(PreferencePage.java:241)
	at org.eclipse.jface.preference.PreferenceDialog.createPageControl(PreferenceDialog.java:1426)
	at org.eclipse.jface.preference.PreferenceDialog$8.run(PreferenceDialog.java:1193)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.ui.internal.JFaceUtil.lambda$0(JFaceUtil.java:44)
```
`Discord Bot` is one of many projects in my workspace, and as the error message says, I have closed it.

**Comment by KazeJiyu** [comment]
@Derkades the plug-in is not disturbed by closed projects. Your error has likely been logged when you checked the Properties of your closed project. I'll handle this case in a better way in the future but it cannot be the source of the failure, and besides, I successfully managed to use the plug-in in a workspace containing several closed projects.

I will deploy a new release embedding a _debug mode_ which should help me to find out what the problem is.

**Comment by Clem-Fern** [comment]
I do not have the problem anymore and I have neither change the version of eclipse, nor that of windows.
So it's ok for me

**Comment by OmgItzValk** [comment]
@Clem-Fern I still have your problem but I'm confused because I don't have any errors, you sure you did nothing?

**Comment by KazeJiyu** [comment]
If no error is reported to the _Error Log_ then problem might come from Discord RCP's SDK ; the incoming _debug mode_ should allow me to determine this more easily.

I do not have the time to work on it right now but it should be done by Sunday.

**Comment by Derkades** [comment]
I had to reinstall Windows because my installation got corrupt. Also reinstalled eclipse and this plugin, and it worked. I'm not sure which of the three fixed it though

**Comment by KazeJiyu** [comment]
Thank you for the feedback, I am happy to know that the issue is fixed for you! Sadly I really don't have the time to work on this project at the moment so I cannot help the others.

**Comment by SkytAsul** [comment]
I'm having the same problem : nothing in Error Log, using the latest version... Do you think that comes from the using of the beta version of Discord ? (Discord PTB)

**Comment by KazeJiyu** [comment]
It is a possibility, but I don't think so. I am still very busy but I will try to release as soon as possible a new version embedding more debug logs and the latest version of [java-discord-rcp](https://github.com/MinnDevelopment/java-discord-rpc) (the library I used to interact with Discord) which should help to fix the issue.

**Comment by KazeJiyu** [comment]
So I finally took the time to add some logs. I deployed a new (unofficial) 1.0.2 version that can be downloaded from [https://dl.bintray.com/kazejiyu/eclipse-discord-integration/snapshots/1.0.2](https://dl.bintray.com/kazejiyu/eclipse-discord-integration/snapshots/1.0.2) (with `Help > Install new software...`).

This version embeds a new "debug mode". This mode can be activated from the preferences (`Window > Preferences > Discord Rich Presence`):

![image](https://user-images.githubusercontent.com/25926433/53209754-c950d900-363b-11e9-972c-9d8fcc49bdea.png)

Once this mode is activated you have to open the `Error Log` view (`Window > Show View > Other... > Error Log`). You should then see a bunch of logs being written as you open new files:

![image](https://user-images.githubusercontent.com/25926433/53209939-7cb9cd80-363c-11e9-979c-1e1419bbae53.png)

All these logs can be exported in a single `.log` file by pressing the "Export Log" button on the top right corner of the view. You can use it to share them with me. I hope it will help to find the cause of the issue.

This version also embeds the last `java-discord-rpc`, so it might fix the issue.

**Comment by SkytAsul** [comment]
Nice it works !! Even on Discord PTB + BetterDiscord 😄 
Thank you !

**Comment by KazeJiyu** [comment]
I'm glad to know it! I'll keep the issue open until an official release is deployed.

**Comment by KazeJiyu** [comment]
Should be fixed as of the new `1.0.3` release. Please leave a comment if on this thread if that's not the case so that I can re-open the issue.
